api_specification/api_specification.json

```diff
@@ -0,0 +1,91 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "description" : "The api starts with `/v1/<contract-category>/`\neach category will have it's interface.\n- The only required parameters for `DIGITS` are `shortcode` + `currency`.\n- We have explicitly decided not to support back pricing in this endpoint.\n- A future endpoint will have an extra date_pricing parameter, to support back pricing.\n",
+    "version" : "1.0.0",
+    "title" : "Pricing Endpoint"
+  },
+  "host" : "virtserver.swaggerhub.com",
+  "basePath" : "/weisheng-binary/test/1.0.0",
```

Comment:
Should probably expect to vary the host, but the basepath to be the same
Note: Semver is in the version here.